src/modules/jcms_migrate/src/Plugin/migrate/process/JCMSCollectionContent.php

```diff
@@ -20,6 +20,12 @@ class JCMSCollectionContent extends AbstractJCMSContainerFactoryPlugin {
   public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
     if (!empty($value)) {
       if (!isset($this->configuration['multiple']) || $this->configuration['multiple'] === FALSE) {
+        if (is_string($value)) {
+          if (strpos($value, '{') === FALSE) {
```

Comment:
The expectation if it is a string is that it is json but when preparing the expression in the sql query it was stripping out the curly brackets. So, if the string doesn't start with `{` and it is a string then I wrap it in curly brackets.

The expression is here:
https://github.com/elifesciences/journal-cms/pull/40/files#diff-715cd3ad4888a4dc0889bb60b0cf12a3R32